responses/90-whats-next.md

```diff
@@ -8,13 +8,13 @@ As a resource for later, here are links to the content you just completed
 
 - :mag: [Debugging in a pipeline]({{ store.debug_url }})
 - :eyes: [Visualizing pipeline dependencies]({{ store.visualize_url }})
-- :speech_balloon: [`which_dirty()` and `why_dirty()`]({{ store.why_dirty_url }}) to further interrogate the status of pipeline targets
+- :speech_balloon: [`tar_visnetwork()` and `tar_outdated()`]({{ store.why_dirty_url }}) to further interrogate the status of pipeline targets
 - :arrows_clockwise: [Cyclical dependencies]({{ store.cyclical_url }})
 - :warning: [Undocumented file output from a function]({{ store.undoc_file_url }})
 - :open_file_folder: [Using a directory as a dependency]({{ store.dir_targets_url }})
-- :clipboard: [Understanding the `I()` helper]({{ store.inhibit_url }})
-- :unlock: [The `target_name` special variable]({{ store.target_name_url }})
-- :gift: [Different kinds of pipeline targets]({{ store.target_kinds_url }}), including [objects]({{ store.object_targets_url }}) and [files]({{ store.file_targets_url }})
+- :clipboard: [Different kinds of pipeline targets]({{ store.target_kinds_url }}), including [objects]({{ store.object_targets_url }}) and [files]({{ store.file_targets_url }})
+- :gear: [Conventions followed in USGS Data Science pipelines]({{ store.target_conventions_url }})
+- :unlock: [`.gitignore` and commenting pipeline code]({{ store.final_tips_url }})
 
 
 :keyboard: Activity: If you have comments or questions, add them below and then assign a course lead this issue to engage in dialogue. When you are satisfied with the conversation, close this issue.
```

Comment:
It might be nice to leave this issue open for easy access to that handy list of resources and links above. I forget - does Learning Lab require that you close this issue for the course to be considered completed? Does it matter to us if Learning Lab calls it completed or not?

Reply:
It doesn't allow you to complete without you closing because [closing the issue is the final action in the final step](https://github.com/USGS-R/ds-pipelines-targets-2-course/blob/main/config.yml#L201-L202). We might be able to delete that so that it calls the course complete beforehand?